### PR TITLE
update website for 1.5.0

### PR DESCRIPTION
**Pull request by roywei**
1. updated installation image
2. updated versions
3. update 1.4.0 to 1.4.1 which should be done in last release. 

@aaronmarkham @IvyBazan @szha @eric-haibin-lin

### CODE REVIEW

**Comment by IvyBazan** [comment]
Here's a site build preview of the latest changes: 
http://100.24.115.165/

It seems the site nav is still broken.

**Comment by roywei** [comment]
Thanks @IvyBazan  It seems the installation is fixed and updated.
This PR is not intended to fix Nav bar and I have no idea how to fix it yet.

**Comment by IvyBazan** [comment]
@roywei @aaronmarkham I've updated the preview link to reflect a full site build with 1.5.0 as the new master

**Comment by roywei** [comment]
Hi @IvyBazan  are you sure it's updated? 
I have made sure there is not 1.4.0 in the doc any more, removed [all lines](https://github.com/apache/incubator-mxnet/pull/15641/files#diff-9a6e753a2d4579af9a1100510b4fc28bL132) saying:
```
pip install mxnet==1.4.0
```

But somehow it still show up on stage website?

File: docs/install/index.md [hunk 1]
```diff
@@ -285,17 +296,24 @@ To build from source, refer to the <a href="ubuntu_setup.html">MXNet Ubuntu inst
 
 <div class="gpu">
 <div class="pip">
-<div class="v1-4-0">
+<div class="v1-5-0">
```
Comment (aaronmarkham) [inline docs/install/index.md]: This is the second mention of v1-5-0 and will mean more stuff will appear.

File: docs/install/index.md [hunk 2]
```diff
@@ -285,17 +296,24 @@ To build from source, refer to the <a href="ubuntu_setup.html">MXNet Ubuntu inst
...
 
 ```
-$ pip install mxnet-cu92
+$ pip install mxnet-cu101
 ```
 
-</div> <!-- End of v1-4-0 -->
+</div> <!-- End of v1-5-0 -->
+<div class="v1-4-1">
```
Comment (aaronmarkham) [inline docs/install/index.md]: Second mention of v1-4-1 ...

File: docs/install/index.md [hunk 3]
```diff
@@ -599,12 +617,18 @@ Refer to the <a href="ubuntu_setup.html#install-the-mxnet-package-for-perl">Perl
...
 <div class="pip">
-<div class="v1-4-0">
+<div class="v1-5-0">
 
 ```
 $ pip install mxnet
 ```
-</div> <!-- End of v1-4-0 -->
+</div> <!-- End of v1-5-0 -->
+<div class="v1-4-1">
```
Comment (aaronmarkham) [inline docs/install/index.md]: Third mention...

File: docs/install/index.md [hunk 4]
```diff
@@ -892,13 +916,20 @@ For more installation options, refer to the <a href="osx_setup.html">MXNet macOS
...
-<div class="v1-4-0">
+<div class="v1-5-0">
 
 ```
 $ pip install mxnet
 ```
 
-</div> <!-- End of v1-4-0 -->
+</div> <!-- End of v1-5-0 -->
+<div class="v1-4-1">
```
Comment (aaronmarkham) [inline docs/install/index.md]: 4th mention...

**Comment by roywei** [comment]
I still don't understand how multiple mentions of versions in <div> will break things. I checked all <div> has propper closing </div>.  Although the parsing may not be robust, but the problem is `pip install mxnet==1.4.0` is showing up even if I removed all appearance of such text.  
Since such text only appears and mess up installation page only when build all versions (v1.5.0 stand alone is fine), the only reason could be website code is wrong in v1.4.0 and the text is checked out from here: https://github.com/apache/incubator-mxnet/blob/master/docs/build_version_doc/build_all_version.sh#L159

Given that, I think it will take longer time for me to figer out the root cause of the bug. I'd like to separate address this issue. I have removed related item in description.

**Comment by abhinavs95** [comment]
@mxnet-label-bot add [Website]

**Comment by ChaiBapchya** [comment]
the reason why
```
$ pip install mxnet==1.4.0
```
continues to exist is because for some weird reason
only that particular div class = “v1-4-0” doesn’t have style=“display: none;” 

Screen Shot 2019-07-26 at 1 09 22 PM

If somehow, we are able to set the CSS using JS, we can get rid of this 1.4.0 issue.

I tried to see the onclick event for buttons in the version related dropdown. but it looks generic and hence can’t seem to single out why only v1.4.0 div isn’t getting CSS.

**Comment by zhreshold** [comment]
I've double checked http://100.24.115.165/install/index.html?platform=Linux&language=Python&processor=CPU there's no 1.4.0 anymore, am I missing something?

**Comment by ChaiBapchya** [comment]
Yes. As mentioned in other duplicate PR #15673 
For version 1.5.0
http://100.24.115.165/versions/1.5.0/install/index.html?platform=Linux&language=Python&processor=CPU

**Comment by aaronmarkham** [comment]
You still need settings.ini updated but let's do that in a follow-up. These changes are good.

**Pull request merged**